pages/index.js

```diff
@@ -11,99 +14,125 @@ import IndexIsssueListItem from "../components/shared/IndexIssueListItem";
 function ServiceItem(props) {
   const { icon, content } = props;
   return (
-    <Col md={3} className="pb-4">
+    <Col className="px-3">
       <Row>
-        <Col xs="auto" className="pe-0">
-          <span style={{ color: "#00bfa5" }}>
-            <h1>{icon}</h1>
-          </span>
-        </Col>
-        <Col>
-          <h5>{content}</h5>
-        </Col>
+        <h1>{icon}</h1>
+      </Row>
+      <Row>
+        <p style={{ fontSize: "1.1rem" }}>{content}</p>
       </Row>
     </Col>
   );
 }
 
 export default function HomeView() {
-  const { projectIssues: projects } = useContext(IssuesContext);
+  const { issues } = useContext(IssuesContext);
+
+  const featuredIssues = [];
+
+  issues.map((issue) => {
+    if (issue.isFeatured) {
+      featuredIssues.push(issue);
```

Comment:
We used to use only projects for the featured tab, now we use both products and projects so I import all issues and create a featuredIssues array based on the label.